core-data/model-resolvers.js

```diff
@@ -0,0 +1,19 @@
+/**
```

Comment:
Should we organize models-related files into their own folder if there's several related ones which stand apart from the rest of the files?

Reply:
Move those to the `index.js`. It's not easy to find a good location without creating cycle dependencies.